### PR TITLE
Release v2.0.4

### PR DESCRIPTION
**Pull request by bitcartel**

### CODE REVIEW

File: doc/authors.md [hunk 1]
```diff
@@ -50,8 +50,10 @@ Per Grön (3)
 Patick Strateman (3)
 Jason Davies (3)
 James O'Beirne (3)
+Gareth Davies (3)
 Daniel Kraft (3)
 Alfie John (3)
+zebambam (2)
 rofl0r (2)
 paveljanik (2)
```
Comment (daira) [inline doc/authors.md]: Should be an alias for Pavel Janik. (Does not block.)

File: doc/authors.md [hunk 2]
```diff
@@ -50,8 +50,10 @@ Per Grön (3)
 Patick Strateman (3)
 Jason Davies (3)
 James O'Beirne (3)
+Gareth Davies (3)
 Daniel Kraft (3)
 Alfie John (3)
+zebambam (2)
```
Comment (daira) [inline doc/authors.md]: This should be an alias for Benjamin Winston. (Does not block.)

File: contrib/debian/changelog [hunk 3]
```diff
@@ -1,3 +1,9 @@
+zcash (2.0.4) stable; urgency=medium
+
+  * 2.0.4 release.
+
+ -- Zcash Company <team@z.cash>  Tue, 26 Mar 2019 11:29:21 -0700
```
[the remark this answers is not in the document]
Comment (bitcartel) [reply contrib/debian/changelog]: I pushed a commit to change this.

**Comment by bitcartel** [comment]
@zkbot r+

**Pull request merged**